os/fs/driver/mtd/smart.c

```diff
@@ -503,6 +503,28 @@ static const struct block_operations g_bops = {
  * Public Functions
  ****************************************************************************/
 
+#ifdef CONFIG_DEBUG_FS
+static int smart_dumpsector(FAR struct smart_struct_s *dev, uint16_t physsector)
```

Comment:
Please add description for this function similar to other functions in this file

Reply:
added.